src/contracts/Pool.ts

```diff
@@ -153,4 +154,13 @@ export class Pool extends BaseContract<typeof poolAbi> {
         }),
     );
   }
+
+  async getMaxOfferDuration() {
+    const maxQueues = await this.contract.read.getMaxTotalWithdrawalQueues();
+    const getMinTimeBetweenWithdrawalQueues =
+      await this.contract.read.getMinTimeBetweenWithdrawalQueues();
+    const maxOfferDurationSeconds =
+      maxQueues * getMinTimeBetweenWithdrawalQueues - BigInt(daysToSeconds(7));
```

Comment:
What's this 7 value? Will it be hardcoded in the contract?

Reply:
It's hardcoded in the contract, yep